tracpro/templates/polls/poll_read.html

```diff
@@ -109,10 +113,73 @@
           </tr>
         </tbody>
       </table>
+
+    {% elif question.question_type == question.TYPE_MULTIPLE_CHOICE and question.pollrun_dict %}
+      <div id='chart_{{ question.pk }}' class='chart' data-window-max='{{ window_max }}' data-window-min='{{ window_min }}' ></div>
+      <script type='text/javascript'>
+        $(function () {
+          $('#chart_{{ question.pk }}').highcharts({
+              chart: {
+                  type: 'area'
+              },
+              title: {
+                  text: ''
+              },
+              xAxis: {
+                  categories: [
+                            {% for date in question.date_list %}
+                              '{{ date }}'{% if not forloop.last %},{% endif %}
+                            {% endfor %}
+                            ],
+                  tickmarkPlacement: 'on',
+                  title: {
+                      enabled: false
+                  }
+              },
+              yAxis: {
+                  title: {
+                      text: 'Percent'
+                  }
+              },
+              tooltip: {
+                  pointFormat: '<span style="color:{series.color}">{series.name}</span>: <b>{point.percentage:.1f}%</b> ({point.y:,.0f})<br/>',
+                  shared: true
+              },
+              plotOptions: {
+                  area: {
+                      stacking: 'percent',
+                      lineColor: '#ffffff',
+                      lineWidth: 1,
+                      marker: {
+                          lineWidth: 1,
+                          lineColor: '#ffffff'
+                      }
+                  }
+              },
+              series:
+              [
+              {% for category, count_list in question.pollrun_dict.items %}
+              {
+                name: '{{ category }}',
+                data: {{ count_list }}
+              },
+              {% endfor %}
+              ]
+          });
+        });
+      </script>
+
+    {% elif question.question_type == question.TYPE_OPEN and question.pollrun_dict|length > 2 %}
+      <!-- word cloud chart -->
+      <div id='chart_{{ question.pk }}' class='chart' ></div>
+      <script type="text/javascript">
+        var words = {{ question.pollrun_dict|safe }};
```

Comment:
Yeah, that makes sense. I guess this wasn't an issue when it was in the coffee script.